tensorflow/examples/tutorials/word2vec/word2vec_basic.py

```diff
@@ -197,13 +208,27 @@ def generate_batch(batch_size, num_skips, skip_window):
   similarity = tf.matmul(
       valid_embeddings, normalized_embeddings, transpose_b=True)
 
+  # Merge all summaries.
+  merged = tf.summary.merge_all()
+
   # Add variable initializer.
   init = tf.global_variables_initializer()
 
+  # Create a saver.
+  saver = tf.train.Saver()
+
 # Step 5: Begin training.
 num_steps = 100001
 
+log_path = 'your_full_path/log'
```

Comment:
Would it be possible to have a flag for the log directory? Sort of like [mnist_with_summaries.py](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/tutorials/mnist/mnist_with_summaries.py)?

Reply:
I have made the changes, thanks for the comment.

Reply:
I can't see the changes, could you check?

Reply:
I've checked but I couldn't find the problem. The last commit for log directory changes: https://github.com/tensorflow/tensorflow/pull/14908/commits/7d18efae4d355713f9dc600814dad73d2aeccef5